frontend/readcollection.lua

```diff
@@ -3,15 +3,17 @@ local LuaSettings = require("luasettings")
 local getFriendlySize = require("util").getFriendlySize
 local lfs = require("libs/libkoreader-lfs")
 local realpath = require("ffi/util").realpath
-local util = require("frontend/util")
+local util = require("util")
 
+local DEFAULT_COLLECTION_NAME = "favorites"
 local collection_file = DataStorage:getSettingsDir() .. "/collection.lua"
 
 local ReadCollection = {}
 
-function ReadCollection:read(collections)
-    local collection_handle = LuaSettings:open(collection_file)
-    local coll = collection_handle:readSetting(collections) or {}
+function ReadCollection:read(collection_name)
+    if not collection_name then collection_name = DEFAULT_COLLECTION_NAME end
```

Comment:
Because all that functions begin with 
`local coll = self:read(collection_name)` 
so we don't need to use
`if not collection_name then collection_name = DEFAULT_COLLECTION_NAME end` 
second time.